examples/seismic/model.py

```diff
@@ -55,6 +56,25 @@ def demo_model(preset, **kwargs):
                             shape=shape, dtype=dtype, spacing=spacing, nbpml=nbpml,
                             **kwargs)
 
+    if preset.lower() in ['constant-viscoelastic']:
+        # A constant single-layer model in a 2D or 3D domain
+        # with velocity 2.2km/s.
```

Comment:
Should the comments reflect a constant value?
What I mean is that maybe it is better to have:

        # A constant single-layer model in a 2D or 3D domain
        # with velocity `vp` km/s.

Reply:
Velocity is fine here, people will know you're referring to `vp` if you don't specify.